src/6_branch.js

```diff
@@ -462,6 +582,15 @@ Branch.prototype['init'] = wrap(
 									journeyInTestMode
 								);
 							}
+							// TODO: check for Twitter referrer, try to use a decision from backend to make this change dynamic without web sdk
+							else if (!branchMatchIdFromOptions && utils.getParamValue('branchify_url') && self._referringLink()) {
+								var linkOptions = {
+									make_new_link: false,
+									open_app: true
+								};
+								this['branch']['deepview']({}, linkOptions);
+								journeys_utils.branch._publishEvent('willNotShowJourney');
```

Comment:
how can the backend avoid publishing a cta view in this case?

Reply:
working on a way to send `auto_branchify=true` in the outgoing v1/deepview request and not published if true: https://github.com/BranchMetrics/branch-backend/pull/5097/files